README.md

```diff
@@ -41,7 +41,7 @@ Developers using the CSS files should keep the directory structure as is, so the
 
 ```scss
 $font-prefix: './custom/path/to/font/files';
-@import 'node_modules/@ibm/plex/scss/ibm-plex.scss';
+@use 'node_modules/@ibm/plex/scss/ibm-plex.scss';
```

Comment:
I'm not sure if this will work because `@use` declarations are required to be the first thing in the file. Defining the `$font-prefix` variable first will have no impact on the path provided to `ibm-plex.scss`.

We may need to reconfigure how the scss is generated to support sass module configuration using `with`

```
@use 'node_modules/@ibm/plex/scss/ibm-plex.scss' with (
  $font-prefix: './custom/path/to/font/files';
);
```

Reply:
To be clear, I don't think the above will work with the way the scss is structured right now